infrastructure/network/ingress.yaml

```diff
@@ -176,24 +176,36 @@ spec:
                 name: api-admin
                 port:
                   number: 80
-          - path: /demo
+          - path: /chatplugin.authenticate
             pathType: Prefix
             backend:
               service:
-                name: frontend-demo
+                name: sources-chatplugin
                 port:
                   number: 80
-          - path: /chatplugin.authenticate
+          - path: /chatplugin.send
             pathType: Prefix
             backend:
               service:
                 name: sources-chatplugin
                 port:
                   number: 80
-          - path: /chatplugin.send
-            pathType: Prefix
+    - host: "demo.airy"
+      http:
+        paths:
+          - path: /
             backend:
               service:
-                name: sources-chatplugin
+                name: frontend-demo
+                port:
+                  number: 80
+    - host: "assets.airy"
+      http:
+        paths:
+          - path: /
+            backend:
+              service:
+                name: frontend-chat-plugin
```

Comment:
I get the `assets.airy` (after all I suggested the dns name) but when you read it here it feels strange that the chat plugin has )1 its own pod 2) "serves" all the content behind assets (I get the second point is "temporary")

Reply:
Very fair!